rackhd_stub.go

```diff
@@ -0,0 +1,365 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+	"net/http/httptest"
+)
+
+func TagsHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	_, err := w.Write([]byte(`["ceph-mon","ceph-node","new"]`))
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+
+		return
+	}
+
+	return
+}
+
+func TagsNodesHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	w.Header().Set("Content-Type", "application/json")
+	if vars["tag"] == "ceph-node" {
+		_, err := w.Write([]byte(`[
```

Comment:
put this strings into constants. you can create a separate file, something like `fixtures.go` and have the constants there